org.opent2t.sample.thermostat.superpopular/com.insteon.thermostat/js/thingTranslator.js

```diff
@@ -193,7 +192,7 @@ function providerSchemaToPlatformSchema(providerSchema, expand) {
                 icv: "core.1.1.0",
                 dmv: "res.1.1.0",
                 rt: ['opent2t.d.thermostat'],
-                di: thermostatDeviceDi,
+                di: generateGUID( providerSchema['DeviceID']+'opent2t.d.thermostat'),
```

Comment:
this.controlId is not available here.

---
In reply to: [108803732](https://github.com/openT2T/translators/pull/217#discussion_r108803732) [](ancestors = 108803732)